generated/nixnetsocket/nixnetsocket.proto

```diff
@@ -297,7 +306,10 @@ message GetNameInfoRequest {
   SockAddr addr = 2;
   int32 host_len = 3;
   int32 serv_len = 4;
-  int32 flags = 5;
+  oneof flags_enum {
```

Comment:
I updated this to use the `GetNameInfoFlags` enum. I only saw these used one at a time (no combination) in [GetAddrNameInfoTests](https://dev.azure.com/ni/DevCentral/_git/ni-central?path=%2Fsrc%2Fxnet%2FxnetCored%2Fnixntipstack_tests%2Fsource%2Fnixntipstackt%2FGetAddrNameInfoTests.cpp&_a=contents&version=GBmain) so I kept them as just the enum.

For this one, I could potentially mark it in functions.py as a 'bitfield_as_enum_array`. I think currently we only support that for output parameters but I could do some work to get it generating for input params as well.

Reply:
@astarche , for this one since it's on a request message (in functions.py) I'm assuming we'll address it with AB#1896770 which will add support for `bitfield_as_enum_array` on input params. Is how I updated the field in this PR okay for now or do you want me to make any further updates before we get to that task?